src/components/feedback/feedbackWidget.tsx

```diff
@@ -0,0 +1,78 @@
+import React, {useCallback, useEffect, useState} from 'react';
+
+import {FeedbackButton} from './feedbackButton';
+import {FeedbackModal} from './feedbackModal';
+import {FeedbackSuccessMessage} from './feedbackSuccessMessage';
+import {sendFeedbackRequest} from './sendFeedbackRequest';
+
+const replay = window.Sentry.getCurrentHub()
+  .getClient()
+  ?.getIntegration(window.Sentry.Replay);
+
+async function sendFeedback(data, replayId, pageUrl): Promise<Response | null> {
+  const feedback = {
+    message: data.comment,
+    email: data.email,
+    replay_id: replayId,
+    url: pageUrl,
+  };
+  const response = await sendFeedbackRequest(feedback);
+  return response;
+}
+
+export function FeedbackWidget() {
+  const [open, setOpen] = useState(false);
+  const [showSuccessMessage, setShowSuccessMessage] = useState(false);
+
+  useEffect(() => {
+    if (!showSuccessMessage) {
+      return () => {};
+    }
+    const timeout = setTimeout(() => {
+      setShowSuccessMessage(false);
+    }, 6000);
+    return () => {
+      clearTimeout(timeout);
+    };
+  }, [showSuccessMessage]);
+
+  const handleSubmit = (data: {comment: string; email: string; name: string}) => {
+    // Prepare session replay
+    replay.flush();
+    const replayId = replay.getReplayId();
+
+    const pageUrl = document.location.href;
+
+    sendFeedback(data, replayId, pageUrl).then(response => {
+      if (response) {
+        setOpen(false);
+        setShowSuccessMessage(true);
+      } else {
+        // eslint-disable-next-line no-alert
+        alert('Error submitting your feedback. Please try again');
+      }
+    });
+  };
+
+  const handleKeyPress = useCallback(event => {
+    // Shift+Enter
+    if (event.shiftKey && event.keyCode === 13) {
```

Comment:
we should add this keyboard access feature to the draft docs

Reply:
Hmm not sure we should even have this imo

Reply:
removed